.github/workflows/dsm-test.yml

```
@@ -98,27 +99,26 @@ jobs:
       # In this next section (only launched if the visual regression failed) we will update the visual snapshots
       # to provide a pull request to the developer contributing to the DSM.
       - name: Remove files generated by previous steps
-        if: failure() && steps.visual_test_execution.outcome == 'failure'
+        if: failure() && steps.visual_test_execution.outcome == 'failure' && steps.example_integration_tests.outcome == 'success'
```

```suggestion
        if: failure() && steps.visual_test_execution.outputs.status == 'failure' && steps.example_integration_tests.outputs.status == 'success'
```

This is the objective to add 
`echo ::set-output name=status::success`
no ?

---

yes